static/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -47,3 +47,20 @@ The performance of NTFS degrades while handling large volumes of files in a
 directory.
 [Here](https://stackoverflow.com/questions/197162/ntfs-performance-and-large-volumes-of-files-and-directories)
 is the resource for reference.
+
+## Enabling paging with `less`
+
+By default DVC uses `less` to page output for some commands. Although, many
```

Comment:
Should we mention here that less is just one of the options and can be overridden with DVC_PAGER?

Reply:
Maybe a link to https://github.com/iterative/dvc.org/pull/831/files#diff-20c0689b083dedb8ffeb8a05226dd834R66 but in the docs?

Reply:
link to the command reference - your section about DVC_PAGER I think should be enough, right?

Reply:
sounds good and done